python/ctsm/test/test_sys_mesh_modifier.py

```diff
@@ -96,8 +96,8 @@ def setUp(self):
         self._lon_dimname = fsurdat_in_data[self._lat_varname].dims[1]
 
         ncap2_cmd = (
-            "ncap2 -A -v -s 'mod_lnd_props=PFTDATA_MASK' "
-            + "-A -v -s 'landmask=PFTDATA_MASK' "
+            "ncap2 -A -v -s 'mod_lnd_props=LANDFRAC_PFT' "
```

Comment:
@slevis-lmwg I had to change this because we decided to no longer put PFTDATA_MASK on the surface datasets in ctsm5.2. This works this way, but I could either use LANDFRAC_PFT or LANDFRAC_mksurfdata do you see any reason to prefer one field over the other?

Reply:
I feel more familiar with LANDFRAC_PFT, so I'm leaning in favor of that.

Reply:
@slevis-lmwg so this isn't working for me now. So I'd like to have you and I look at it together. I also realized that this test doesn't effectively do anything because it's not changing the mask. I think we should have a test that does change the mask and make sure it's correct.

Reply:
@slevis-lmwg and I went over this yesterday and worked on it together. It helped that we worked together, but we didn't find a solution. But, there were several things that we noticed looking at it together, and I'll work on it more, and then get back with Sam if I'm stuck.

Reply:
I got this working now, by both converting to integer earlier in the process. And finding that the error we saw was in a different place in the code. I'm close on that one now.

Reply:
Working with the latest commit. But, we should still add a test that changes the mesh and verifies the change is correct.

Reply:
> Erik, I can check when I arrive in Colorado:  I wonder whether we have such
test(s) in the unit tests.

From what I can tell, we do not...